cmake/Version.cmake

```diff
@@ -27,7 +27,7 @@ set(DATAFED_FOXX_API_PATCH 0)
 
 set(DATAFED_WEB_MAJOR 1)
 set(DATAFED_WEB_MINOR 0)
-set(DATAFED_WEB_PATCH 0)
```

Comment:
Version numbers need to be updated.

Reply:
COMMON lib patch update, fixes to log output
COMMON PROTOBUF minor update
FOXX API and FOXX minor update addition of getters and setters associated with tokens
WEB minor update  - updates to transfer dialogue. 
AUTHZ lib patch update - fix log out put. 
Python client no longer beta.